### ISSUE TITLE
Root mount script clobbering occurs due to backslash escaping issues

### ISSUE THREAD

**Issue by Ittai-Dokoni** [reisxd/revanced-builder#490]
I've had issues with rooted YT installs crashing upon boot. Checking the Magisk mount script I see that that due to escaping issues it's clobbering the mount script during the install:

```
flame:/data/adb/service.d # cat mount_revanced_com.google.android.youtube.sh
#!/system/bin/sh
')" != "1" ]; do sleep 1; donet_completed | tr -d '
base_path="/data/adb/revanced/com.google.android.youtube.apk"
    stock_path=$(pm path com.google.android.youtube | grep base | sed 's/package://g')
    chcon u:object_r:apk_data_file:s0 $base_path
    mount -o bind $base_path $stock_pathflame:/data/adb/service.d #
```

**Comment by SimonvBez** [comment]
> I believe the issue is writeFileSync is interpreting '\r', and therefore needs to have \ escaped (\\).

The carriage return (\r) is performed by `cat` when printing the file's content to the terminal.
When you open the file with a text editor like nano you can see that the full line is stored correctly as
`    while [ "$(getprop sys.boot_completed | tr -d '^M')" != "1" ]; do sleep 1; done`

So that's not the cause of your issue. It does seem however that the mount script you have is outdated. It should have two lines about `MAGISKTMP` and `MIRROR` as seen in https://github.com/reisxd/revanced-builder/blob/main/utils/mountReVanced.js#L42
Could it be you're running an outdated version of the builder?

**Comment by shrihanDev** [comment]
Could be fixed by the fix for #443, so closing.
